bees/twiliobee/twiliobeefactory.go

```diff
@@ -0,0 +1,107 @@
+package twiliobee
+
+import (
+	"github.com/muesli/beehive/bees"
+)
+
+// TwilioBeeFactory is a factory for TwilioBees.
+type TwilioBeeFactory struct {
+	bees.BeeFactory
+}
+
+// New returns a new Bee instance configured with the supplied options.
+func (factory *TwilioBeeFactory) New(name, description string, options bees.BeeOptions) bees.BeeInterface {
+	bee := TwilioBee{
+		Bee: bees.NewBee(name, factory.ID(), description, options),
+	}
+	bee.ReloadOptions(options)
+
+	return &bee
+}
+
+// ID returns the ID of this Bee.
+func (factory *TwilioBeeFactory) ID() string {
+	return "twiliobee"
+}
+
+// Name returns the name of this Bee.
+func (factory *TwilioBeeFactory) Name() string {
+	return "Twilio"
+}
+
+// Description returns the description of this Bee.
+func (factory *TwilioBeeFactory) Description() string {
+	return "Sends SMS messages"
+}
+
+// Image returns the filename of an image for this Bee.
+func (factory *TwilioBeeFactory) Image() string {
+	return factory.ID() + ".png"
+}
+
+// LogoColor returns the preferred logo background color (used by the admin interface).
+func (factory *TwilioBeeFactory) LogoColor() string {
+	return "#0d122b"
+}
+
+// Options returns the options available to configure this Bee.
+func (factory *TwilioBeeFactory) Options() []bees.BeeOptionDescriptor {
+	opts := []bees.BeeOptionDescriptor{
+		{
+			Name:        "account_sid",
+			Description: "Twilio account SID",
+			Type:        "string",
+			Mandatory:   true,
+		},
+		{
+			Name:        "auth_token",
+			Description: "Twilio auth token",
+			Type:        "string",
+			Mandatory:   true,
+		},
+		{
+			Name:        "from_number",
+			Description: "Phone number to send SMS messages from",
+			Type:        "string",
+			Mandatory:   true,
+		},
+		{
+			Name:        "to_number",
+			Description: "Phone number to send SMS messages to",
+			Type:        "string",
+			Mandatory:   true,
+		},
+	}
+	return opts
+}
+
+// Events describes the available events provided by this Bee.
+func (factory *TwilioBeeFactory) Events() []bees.EventDescriptor {
```

Comment:
Since this method just returns the default value, you can actually remove it altogether.